playground/css-entrypoint/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Would you move the test to playground/legacy by using https://github.com/vitejs/vite/blob/main/CONTRIBUTING.md#:~:text=Some%20playgrounds%20define,assets%20playground ?

Reply:
I moving the test into a new variant of playground/legacy.

The non-watch build on current main also produced a `polyfills-legacy.js` and a `...-legacy.js` for the vite configuration with only a single css chunk. I added them to the tests since I don't want do change this behavior in this PR. However, when generating HTML on the server I currently only inject the legacy entries for the js files + the legacy runtime. Should I also inject the legacy entries for the css files?

Reply:
I guess you need to add the following content (I think `__vite_is_modern_browser` is a internal thing though):
```js
if (!window.__vite_is_modern_browser) System.import("assets/style-only-entry-wrQa-JKU.css");
```

Reply:
My previous message lacks a bit of context: 
- I'm using the modern + legacy output.
- For js entries on the server, I
  - go through the manifest entry as well as every entry that is referenced in `imports` recursively and:
    - add a `<script type=module>` for the file name in `src`
    - add a `<link rel=stylesheet>` for every file name mentioned in `css`
    - add a `<link rel=preload>` for every file name mentioned in `assets`
  - Then I go through the manifest entry with the `-legacy` suffix and all of its `imports`-children, and only add `<script nomodule>` for the file names in `src`.

So I assume that I don't need to include the legacy-css files, since I've already generated `<link rel=stylesheet>` tags for the styles during the modern run?

Reply:
In that case, it should be fine without it.